README.md

```diff
@@ -110,6 +110,14 @@ Missing from the above list is one final, advanced, trait: `HashAndContext` whic
 
 ## Breaking changes
 
+### In 1.2
```

Comment:
My thinking here was to make it a minor release, since it adds the "feature" of `stable_hash(..., version=3)`. 

If you think it would be more appropriate to communicate that as a patch release, I'm not against it, but I think it would technically be a minor release.

Reply:
yeah that's fine, I think it makes sense I was just pointing out that the version in the Project.toml says 1.1.9
https://github.com/beacon-biosignals/StableHashTraits.jl/pull/64/files#diff-72ed386c2a0cd1d23c0968297e70023ed98c22490d146dd89fc91f48369bad4dR4